### ISSUE TITLE
Web UI to edit text stories

### ISSUE THREAD

**Issue by vorburger** [OASIS-learn-study/minecraft-storeys-maker#282]
A Web UI to edit (non-Scratch) [text stories](https://github.com/OASIS-learn-study/minecraft-storeys-maker/blob/develop/storeys/src/main/resources/hello.story) could be fun?

Either a `/story-edit demo` (or something like that) command which gives a URL with a simple fullscreen textbox.

Or using e.g. https://filebrowser.org on `/var/minecraft/config/storeys-web/stories/` (with some auth). To keep it simple with single container, that would either need a process supervisor in the container, or perhaps https://github.com/vorburger/ch.vorburger.exec to launch `filebrowser`?

@edewit thoughts?

**Comment by edewit** [comment]
yeah that would be really cool, could use [monaco](https://microsoft.github.io/monaco-editor/) for that and even include code completion
